aspnet/Gantt/Editing.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
 title: Editing | Gantt | ASP.NET Webforms | Syncfusion
-description: editing
+description: Learn here about editing  in Syncfusion Essential ASP.NET Webforms Gantt Control, its elements, and more.
```

Comment:
Learn here about editing support in Syncfusion Essential ASP.NET Webforms Gantt Control, its elements, and more.